content/en/community/hackathons/usoc23/behind-the-scenes/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Session 03 - Behind the Scenes
+linkTitle: Working with Unikraft
+weight: 5
+summary: |
+  Interacting with Unikraft by modifying configs and writing first lines of codes.
+  Expected time: 100 minutes
+--- 
+
+## Reminders
+
+### Setup
+
+{{< readfile file="/community/hackathons/sessions/setup/content/Setup.md" markdown="true" >}}
+
+### Support Files
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Required-Tools-and-Resources_Support-Files.md" markdown="true" >}}
+
+### linuxu and KVM
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Overview_02.-linuxu-and-KVM.md" markdown="true" >}}
+
+## Overview
+
+### Unikraft Core
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Overview_03.-Unikraft-Core.md" markdown="true" >}}
+
+### Configuring Unikraft - Config.uk
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Overview_05.-Configuring-Unikraft---Config.uk.md" markdown="true" >}}
+
+### The Build System - basics
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Overview_06.-The-Build-System---basics.md" markdown="true" >}}
+
+## Work items
+
+### 01. Tutorial / Reminder: Building and Running Unikraft
+
+Have a look again at how to configure, build and run the [helloworld](https://github.com/unikraft/app-helloworld) application with Unikraft [here](https://unikraft.org/community/hackathons/usoc23/overview).
+
+### 02. Reminder: Make It Speak
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Work-Items_02.-Tutorial:-Make-It-Speak.md" markdown="true" >}}
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Work-Items_02.-KVM-x86_64.md" markdown="true" >}}
+
+### 03. Reminder: Adding Filesystems to an Application
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Work-Items_06.-Tutorial---Reminder:-Adding-Filesystems-to-an-Application-2023.md" markdown="true" >}}
+
+### 04. Give the User a Choice
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Work-Items_07.-Tutorial:-Give-the-User-a-Choice.md" markdown="true" >}}
+
+### 05. Tutorial: Arguments from the Command Line
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Work-Items_08.-Tutorial:-Arguments-from-Command-Line.md" markdown="true" >}}
+
+### 06. Echo-back Server
+
+{{< readfile file="/community/hackathons/sessions/baby-steps/content/Practical-Work_01.-Echo-back-Server.md" markdown="true" >}}
```

Comment:
You might want to copy-paste this file, since there are a few differences that need to be done to it:

* replace `work/01-echo-back/` with `work/06-echo-back`
* remove the reference to `kraft`, there is no documentation on using networking yet
* Modify the `Makefile` to match the new layout (point to `.unikraft/*` instead of `../../*`)